File: src/widgets/testinfowidget.cpp
#include "testinfowidget.h"
#include "ui_testinfowidget.h"

TestInfoWidget::TestInfoWidget(QWidget *parent) :
    QWidget(parent),
    ui(new Ui::TestInfoWidget)
{
    ui->setupUi(this);
}

TestInfoWidget::~TestInfoWidget()
{
    delete ui;
}

File: dcs/client/ui/test_info_widget.py

from PySide6 import QtWidgets

from .test_info_widget_ui import Ui_TestInfoWidget

class TestInfoWidget(QtWidgets.QWidget, Ui_TestInfoWidget):
    def __init__(self, parent=None):
        super().__init__(parent)
        self.setupUi(self)